script-test/playbackstrategies/msestrategytest.js

```diff
@@ -106,66 +111,77 @@ require(
 
           expect(mseStrategy.transitions.canBePaused()).toBe(true);
         });
+      });
 
-        it('canBeginSeek() Transition is true', function () {
+      describe('Load', function () {
+        it('should create a video element and add it to the media element', function () {
           setUpMSE();
 
-          expect(mseStrategy.transitions.canBeginSeek()).toBe(true);
+          expect(playbackElement.childElementCount).toBe(0);
+
+          mseStrategy.load('src', null, 0);
+
+          expect(playbackElement.firstChild).toBe(mockMediaElement);
```

Comment:
As per my above comment, it might be better to assert on what you know is an audio element, rather than trust the test setup to have provided the correct element type,